tests/sentry/incidents/endpoints/test_serializers.py

```diff
@@ -701,6 +703,7 @@ def test_comparison_delta_below(self):
         params["resolveThreshold"] = 10
         params["triggers"][0]["alertThreshold"] = 50
         params["triggers"][1]["alertThreshold"] = 40
+        params["detection_type"] = AlertRuleDetectionType.PERCENT
         serializer = AlertRuleSerializer(context=self.context, data=params, partial=True)
         assert serializer.is_valid(), serializer.errors
         alert_rule = serializer.save()
```

Comment:
in the serializer tests, you could test the validations raise here by wrapping the `serializer.save()` in the pytest raises as well